index.html

```diff
@@ -21,42 +21,41 @@ <h2 class="ui divider horizontal">La méthode Startup d'État</h2>
 
 
 <section id="phases" class="ui vertically divided two columns stackable padded grid container">
+	{% capture investigationDescription %}
+		<p>Un <abbr title="D’un ministère ou d’un opérateur.">agent public</abbr> identifie une friction entre les usagers et son administration.</p>
+		<p>Des circuits administratifs tortueux, des démarches manquant de fluidité… Aidé par l'incubateur, l'agent public vérifie que cette friction concerne un public suffisamment large, qu’un outil numérique pourrait la faire disparaitre, et que les obstacles techniques sont surmontables. Son administration valide le recrutement d'un ou deux développeurs et responsables produits.</p>
```

Comment:
Désolé, je suis off aujourd'hui :) 
A mardi !  Vincent Berdot tel : 06 50 63 65 30

 
      De : Matti Schneider <notifications@github.com>
 À : sgmap/beta.gouv.fr <beta.gouv.fr@noreply.github.com> 
Cc : vincentberdot <vincentberdot@yahoo.fr>
 Envoyé le : Vendredi 25 mars 2016 18h43
 Objet : Re: [beta.gouv.fr] Éclaircissement des descriptions de phases (#111)
   
In index.html:> @@ -21,42 +21,41 @@ <h2 class="ui divider horizontal">La méthode Startup d'État</h2>
>  
>  
>  <section id="phases" class="ui vertically divided two columns stackable padded grid container">
> +	{% capture investigationDescription %}
> +		<p>Un <abbr title="D’un ministère ou d’un opérateur.">agent public</abbr> identifie une friction entre les usagers et son administration.</p>
> +		<p>Des circuits administratifs tortueux, des démarches manquant de fluidité… Aidé par l'incubateur, l'agent public vérifie que cette friction concerne un public suffisamment large, qu’un outil numérique pourrait la faire disparaitre, et que les obstacles techniques sont surmontables. Son administration valide le recrutement d'un ou deux développeurs et responsables produits.</p>
C'était la version précédente, et à la lecture ça passe beaucoup moins bien car on a une incohérence conceptuelle : les autres phases n'ont pas d'exemple aussi précis.—
You are receiving this because you were mentioned.
Reply to this email directly or view it on GitHub